asciinema/commands/cat.py

```diff
@@ -1,23 +1,25 @@
 import sys
+from typing import Any, Dict
 
-import asciinema.asciicast as asciicast
-from asciinema.commands.command import Command
-from asciinema.term import raw
+from .. import asciicast
+from ..config import Config
+from ..term import raw
+from .command import Command
 
 
 class CatCommand(Command):
-    def __init__(self, args, config, env):
+    def __init__(self, args: Any, config: Config, env: Dict[str, str]):
         Command.__init__(self, args, config, env)
         self.filename = args.filename
 
-    def execute(self):
+    def execute(self) -> int:
         try:
-            stdin = open("/dev/tty")
-            with raw(stdin.fileno()):
-                with asciicast.open_from_url(self.filename) as a:
-                    for t, _type, text in a.stdout_events():
-                        sys.stdout.write(text)
-                        sys.stdout.flush()
+            with open("/dev/tty", "wt", encoding="utf-8") as stdin:
```

Comment:
The reason we open /dev/tty here is to get its file descriptor and turn it into raw mode, then _read_ and ignore any terminal answerbacks, so user doesn't get garbage input in their shell prompt. Therefore open mode could (and probably should) be left as read-only.